tests/README.md

```diff
@@ -1,105 +1,59 @@
-`tests/setup/kfctl_openshift.yaml`.  If you want to modify your test run, you
-might want to change those files to get the behavior that you're looking for.
-After you make changes, you will need to rebuild the test image with `make build`.
+### Pre-requisites
 
-If you'd like to run the tests against an instance that already has Open Data Hub installed,
-you set `SKIP_INSTALL=true` and that will cause the test run
-to skip the installation process and will only run the tests.  example: `make run SKIP_INSTALL=true`
+* Logged into an OCP/K8s cluster
+* Valid kubeconfig file (default is `$HOME/.kube/config`)
+* DSPO is already installed (either via ODH or manual)
+* An empty namespace to run the tests in
 
-If you'd like to run the tests against an instance that already has a KfDef created,
-you set `SKIP_KFDEF_INSTALL=true` and that will cause the test run
-to skip the step of creating the default KfDef.  example: `make run SKIP_KFDEF_INSTALL=true`
 
-If you'd like to run a single test instead of all tests, you can
-set the TESTS_REGEX variable `TESTS_REGEX=<name of the test to run>`.  That will
-only run the test that you specify instead of all of the tests.  example: `make run TESTS_REGEX=grafana`
+### Run tests locally
 
-If you have a local instance already running the operator and you'd like to skip that part
-of the install process, you can set `SKIP_OPERATOR_INSTALL=true` and that will bypass installation
-of the operator, but will still install the authentication for the jupyterhub tests.
 
-For other possible configurations, you can look in the Makefile.  There are a set of
-variables at the top that you could change to meet the needs of your particular test run.
+#### Full test suite
 
-## Test Artifacts
-The environment variable `ARTIFACT_DIR` specifies the root directory where all test artifacts should be
-stored for retrieval at the end of a test run. Any files created should be uniquely named to prevent
-a test from overwriting an artifact generated by another test.
+The full test suite will install a DSPA, wait for it to reach ready status, run the test suite, then clean up the DSPA 
+afterwards.
 
-# Running tests manually
+```bash
 
-Manual running of the tests relies on the test
-runner [located here](https://github.com/AICoE/peak).
-See the README.md there for more detailed information on how it works.
+# Adjust the following as needed
+KUBECONFIG_PATH=$HOME/.kube/config # this is usually the default
+TARGET_CLUSTER=...(e.g. https://api.hukhan.dev.datahub.redhat.com:6443, you can retrieve this via `oc whoami --show-server`)
+TARGET_NAMESPACE=dspa
 
-Note when running on a **mac** you may need to do the following:
+git clone git@github.com:opendatahub-io/data-science-pipelines-operator.git ${DSPO_REPO}
 
-```sh
-brew install coreutils
-ln -s /usr/local/bin/greadlink /usr/local/bin/readlink
-```
-
-Make sure you have an OpenShift login, then do the following:
+# Make sure DSPO is already deployed, if not then run: 
+make deploy
```

Comment:
In my installation, OPERATOR_NS is required. Maybe this should be explicitly set in here too.

Reply:
@rimolive - `OPERATOR_NS` should be set/defaulted in the Makefile....
https://github.com/red-hat-data-services/data-science-pipelines-operator/blob/main/Makefile#L54

Reply:
Still requires manual intervention to  make it work, so either document that `odh-applications` must be created previously, or specify the DSPO namespace manually.

Reply:
updated docs